ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/PrecompiledContractConfiguration.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.ethereum.mainnet;
+
+import tech.pegasys.pantheon.ethereum.core.PrivacyParameters;
+import tech.pegasys.pantheon.ethereum.vm.GasCalculator;
+
+public class PrecompiledContractConfiguration {
```

Comment:
Is there any utility to this class beyond holding 2 objects? Could it's consumers take the gas calculator and privacy parameters separately without losing functionality?